src/login/LoginView.js

```diff
@@ -395,16 +395,21 @@ export class LoginView {
 				args.userId && deviceConfig.getByUserId(args.userId))) {
 				// there are no credentials stored for the desired email address or user id, so let the user enter the password
 				this.mailAddress(args.loginWith)
-				// ensure that input fields have been created after app launch
+
+				// TODO ensure that input fields have been created after app launch
 				// if (this.mailAddress._domInput) {
 				// 	this.mailAddress.animate()
 				// }
+
 				// when we pre-fill the email address field we need to delete all current state
 				this.helpText = lang.get('emptyString_msg')
 				this.invalidCredentials = false
 				this.accessExpired = false
 				this.password("")
 
+				const passwordInput = document.querySelector("input[type=password]")
```

Comment:
sorry, why do we query and click? can we use `oncreate` with raf and then `focus()`?

Reply:
good point about the focus, i seem to remember that it wasn't working for me, but it works now so i'm not sure what that's about. we could grab the dom element in oncreate but i just though it was a bit easier to do it here, since we only need it here anyway